### DISCUSSION TITLE
How to invalidate cached old website?

### DISCUSSION THREAD

**Discussion by ortonas** [vercel/next.js#21806] [Help]
I have reworked a website with nextjs, new looks and moved to vercel as well. However, the old website is still cached on a browser.
It's been 5 days since I deployed v2 of website but I still see the old looks

So what is the trick to invalidate old website? Cache-control definatly sounds related but NextJS says it gets overwritten in prod

**Comment by stigkj** [comment]
So the old website was not a Next.js app, right? If the old website had cache headers that said the main HTML should be cached for a while, there is not really anything you can do with that. It depends a little bit on what cache headers you used, though.
